pubspec.lock

```diff
@@ -181,6 +181,14 @@ packages:
       url: "https://pub.dev"
     source: hosted
     version: "2.1.4"
+  web:
+    dependency: transitive
+    description:
+      name: web
+      sha256: dc8ccd225a2005c1be616fe02951e2e342092edf968cf0844220383757ef8f10
+      url: "https://pub.dev"
+    source: hosted
+    version: "0.1.4-beta"
 sdks:
-  dart: ">=3.0.0-0 <4.0.0"
+  dart: ">=3.1.0-185.0.dev <4.0.0"
```

Comment:
I’d like to see this merged as well! However, using a Dart version from the Dev channel for a production-ready application might not be the best approach. It’s generally recommended to stick with the Stable channel. Since this is an older PR, it might also be a good idea to bump up the version